lib/src/modules/conditional.dart

```diff
@@ -293,17 +299,13 @@ class Sequential extends _Always {
   /// The input clocks used in this block.
   final List<Logic> _clks = [];
 
-  @override
-
   /// This sets the Sequential conditions to reset
   // ignore: overridden_fields
   Logic? reset;
 
-  @override
-
-  /// This sets a particular Sequential conditions receiver to resetVal
+  /// This sets a particular Sequential conditions receiver to resetValues
   // ignore: overridden_fields
```

Comment:
It is possible to remove `ignore: overriden_fields`, since `@override` has been removed.